core/slice.go

```diff
@@ -867,6 +887,10 @@ func (sl *Slice) loadLastState() error {
 	phCache := rawdb.ReadPhCache(sl.sliceDb)
 	for key, value := range phCache {
 		sl.phCache.Add(key, value)
+		// Removing the PendingHeaders from the database
+		rawdb.DeletePhCache(sl.sliceDb)
+		rawdb.DeletePendingHeader(sl.sliceDb, key)
+		rawdb.DeletePhCacheTermini(sl.sliceDb, key)
```

Comment:
I think this is an unrelated bug fix? If so, please separate it into its own bugfix commit